src/containers/EditProfileContainer.js

```diff
@@ -16,20 +20,46 @@ const EditProfileContainer = props => {
     myUserInfo,
     uploadProfileImage,
   } = props
+
+  const [username, setUsername] = useState('')
+
+  useEffect(() => {
```

Comment:
Hmm I don't know why this is needed, since current user should already be present via props, and should update accordingly whenever props change.

Reply:
removed!